src/NUnitFramework/framework/Attributes/TestFixtureAttribute.cs

```diff
@@ -192,8 +192,7 @@ public bool Explicit
             get
             {
                 //return Properties.Get(PropertyNames.Category) as string;
-                var catList = Properties[PropertyNames.Category];
-                if (catList == null)
+                if (!Properties.TryGet(PropertyNames.Category, out var catList))
```

Comment:
indexer allocates a new list on query, so return value would never be null anyway, `TryGet` now allows to return fast